xmodule/graders.py

```diff
@@ -472,7 +470,7 @@ def _min_or_none(itr):
 
 class ShowCorrectness:
     """
-    Helper class for determining whether correctness is currently hidden for a block.
+    Helper class for determining the possible status of correctness.
 
     When correctness is hidden, this limits the user's access to the correct/incorrect flags, messages, problem scores,
     and aggregate subsection and course grades.
```

Comment:
It does not seem to be accurate. This class and `ShowAnswer` only provide constants.